x-pack/plugins/index_management/public/application/sections/home/index_list/details_page/details_page.tsx

```diff
@@ -66,6 +69,12 @@ const tabs = [
     ),
   },
 ];
+
+const statsTab = {
+  id: IndexDetailsSection.Stats,
+  name: <FormattedMessage id="xpack.idxMgmt.indexDetails.statsTitle" defaultMessage="Stats" />,
```

Comment:
Elsewhere we spell out statistics. For example, Field Statistics in Discover, and the Statistics tab in the Inspector.  Should we spell it out in this UI as well?